lib/defaults.js

```diff
@@ -0,0 +1,152 @@
+
```

Comment:
Add a comment about this being a generated file.

The indentation is also off. Not important though.